File: M3 Intermediate DSA/src/L7_Arrays_Subarrays/Assignment4.java
package L7_Arrays_Subarrays;

public class Assignment4 {

  public int[] solve(int[] A, int B, int C) {
    int[] ans = new int[C - B + 1];
    int ansIndex = 0;

    for (int index = B; index <= C; index++) {
      ans[ansIndex] =  A[index];
      ansIndex++;
    }

    return ans;
  }

}

File: M3_Intermediate_DSA/L5_Subarrays/assignment4.go
package main

func SolveA4(A []int, B int, C int) []int {
	ans := make([]int, C-B+1)
	ansIndex := 0

	for index := B; index <= C; index++ {
		ans[ansIndex] = A[index]
		ansIndex++
	}

	return ans
}
